test_regress/driver.pl

```diff
@@ -697,6 +697,22 @@ sub new {
     return $self;
 }
 
+sub simbenchmark_filename {
+    my $self = (ref $_[0] ? shift : $Self);
+    return $self->{obj_dir}."/$self->{name}_simbenchmark.dat";
+}
+
+sub init_simbenchmark {
+    my $self = (ref $_[0] ? shift : $Self);
+    # Simulations with simbenchmark enabled append to the same file between runs.
+    # Test files must ensure a clean benchmark data file before executing tests.
+    my $fh = IO::File->new(">".simbenchmark_filename) or die "%Error: $! ".simbenchmark_filename;
+    print $fh "# Verilator simulation benchmark data\n";
+    print $fh "# Test name: ".$self->{name}."\n";
+    print $fh "# Top file: ".$self->{top_filename}."\n";
+    print $fh "# cycles\ttime[s]\n";
```

Comment:
Perhaps this would be better in a .csv format (with .csv extension) - then easy to suck into Excel etc?